eZ/Publish/Core/Persistence/Cache/ContentTypeHandler.php

```diff
@@ -332,8 +332,17 @@ public function createDraft($modifierId, $typeId)
     public function copy($userId, $typeId, $status)
     {
         $this->logger->logCall(__METHOD__, ['user' => $userId, 'type' => $typeId, 'status' => $status]);
+        $copy = $this->persistenceHandler->contentTypeHandler()->copy($userId, $typeId, $status);
 
-        return $this->persistenceHandler->contentTypeHandler()->copy($userId, $typeId, $status);
+        // Clear loadContentTypes() cache as we remove an item from it's collection here.
```

Comment:
The problem is rather that we add an item right?

Reply:
@andrerom Of course you're right. I was thinking if delete an item will have the same issue at the time of writing this.